src/main/java/org/broadinstitute/hellbender/tools/exome/CopyNumberTriStateAllele.java

```diff
@@ -0,0 +1,120 @@
+package org.broadinstitute.hellbender.tools.exome;
+
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFSimpleHeaderLine;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.hmm.CopyNumberTriState;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Enum of possible CNV alleles.
+ */
+@SuppressWarnings("serial")
+public class CopyNumberTriStateAllele extends Allele {
+
+    public static final CopyNumberTriStateAllele REF = new CopyNumberTriStateAllele(CopyNumberTriState.NEUTRAL, "N", true);
+    public static final CopyNumberTriStateAllele DEL = new CopyNumberTriStateAllele(CopyNumberTriState.DELETION, "<DEL>", false);
+    public static final CopyNumberTriStateAllele DUP = new CopyNumberTriStateAllele(CopyNumberTriState.DUPLICATION, "<DUP>", false);
+
+    public static final String ALT_KEY = "ALT";
+    public static final String DEL_VCF_DESCRIPTION = "Represents a deletion with respect to reference copy number";
+    public static final String DUP_VCF_DESCRIPTION = "Represents a duplication with respect to reference copy number";
+
+    public final CopyNumberTriState state;
+
+    /**
+     * List of all alleles as they should be included in VCF output files.
+     * <p>
+     *     This collection is unmodifiable.
+     * </p>
+     */
+    public static final List<CopyNumberTriStateAllele> ALL_ALLELES =
+            Collections.unmodifiableList(Arrays.asList(REF, DEL, DUP));
+
+    /**
+     * All allele list typed as a {@code List<Allele>} as required by some elements in the VCF framework.
+     */
+    @SuppressWarnings("unchecked")
+    public static final List<Allele> PLAIN_ALL_ALLELES =
+            (List<Allele>) (List) ALL_ALLELES;
+
+    /**
+     * List of all alleles as they should be included in VCF output files.
+     * <p>
+     *     This collection is unmodifiable.
+     * </p>
+     * <p>
+     *     Notice that the collection type is {@link #Allele} rather than {@link #CopyNumberTriStateAllele}.
+     *     This is done to be be able to use this list directly with some of the VCF framework class that expect
+     *     this type of lists.
+     * </p>
+     */
+    public static final List<CopyNumberTriStateAllele> ALTERNATIVE_ALLELES =
+            Collections.unmodifiableList(Arrays.asList(DEL, DUP));
+
+    protected CopyNumberTriStateAllele(final CopyNumberTriState state, final String name, final boolean isRef) {
```

Comment:
@vruano are you responding to a comment by @vruano?  I side with @vruano on this one.